bin/servicerouter.py

```diff
@@ -27,85 +29,87 @@
 Usage:
   $ servicerouter.py --marathon http://marathon1:8080 --haproxy-config /etc/haproxy/haproxy.cfg
 
-  The user servicerouter is executed as must have the right to reload haproxy.
+  The user that executes servicerouter must have the permission to reload
+  haproxy.
 
 
 Environment Variables:
   HAPROXY_GROUP
-  Only servicerouter instances which are members of the given group will
-  point to the service. Service routers with the group '*' will collect
-      all groups.
+    The group of servicerouter instances that point to the service.
+    Service routers with the group '*' will collect all groups.
 
   HAPROXY_{n}_VHOST
-  HTTP Virtual Host proxy hostname to catch
-  Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
+    The Marathon HTTP Virtual Host proxy hostname to gather.
```

Comment:
Maybe, you can also explain the mechanism with the {n} on top of the list for all variables?